.github/workflows/build-tool.yml

```diff
@@ -5,7 +5,6 @@ on:
     branches:
       - main
     paths:
-      - "scripts/setup/**"
```

Comment:
Why this line remove ? We need to trigger build-tool rebuild when `scripts/setup/rust-toolchain.toml` and `scripts/setup/rust-tools.txt` changed.

Reply:
Sorry, it's a mistake

Reply:
Never mind, we can add it back next time upgrading toolchain.